Amplitude/Amplitude.m

```diff
@@ -310,13 +319,13 @@ - (void)initializeApiKey:(NSString*) apiKey userId:(NSString*) userId
         return;
     }
 
-    (void) SAFE_ARC_RETAIN(apiKey);
+    SAFE_ARC_RETAIN(apiKey);
```

Comment:
does removing the (void) affect anything? It's an unrelated change that could be important